core/validation/failures.py

```diff
@@ -159,15 +159,15 @@ def to_message(self) -> tuple[str, str, str]:
 
         if sheet == "Funding Profiles":
             row_str = ", ".join(self.row[1:4])
-            project_number = int(self.row[0].split("-")[2])
+            project_number = get_project_number(self.row[0])
```

Comment:
Didn't we figure out a way to make sure this project number always lines up with the number on the sheet when there are missing/deactivated projects? like on the Blackfriars sheet

Reply:
Yet to be implemented but captured here https://dluhcdigital.atlassian.net/browse/SMD-147